pkg/yurthub/filter/interfaces.go

```diff
@@ -59,4 +59,11 @@ type ObjectFilter interface {
 	Filter(obj runtime.Object, stopCh <-chan struct{}) runtime.Object
 }
 
+type FilterManager interface {
```

Comment:
how about rename FilterManager to FilterFinder?

Reply:
done